ui-v2/tests/acceptance/dc/nspaces/index.feature

```diff
@@ -29,9 +37,5 @@ Feature: dc / nspaces / index: Nspaces List
     And I see 1 nspace model with the description "The default namespace"
   Scenario: The default namespace can't be deleted
     Then I see 3 nspace models
-    Then I fill in with yaml
-    ---
-    s: default
-    ---
-    And I click actions on the nspaces
-    Then I don't see delete on the nspaces
+    And I click nspaces.1.actions
+    Then I don't see nspaces.1.delete
```

Comment:
As the default namespace is second in the listing (due to the new alpha sorting) make sure we interact with the second row (rows are numbered with a zero based index)